File: python/connect.py
# -*- coding: UTF-8 -*-
import sys
import MySQLdb

def main():
	try:
		connect = MySQLdb.connect(db = "test",
						host = "localhost",
						user = "root",
						passwd = "123456")
		# Non-local TCP/IP 
		'''
		connect = MySQLdb.connect(db = "test",
						host = "mysql.example.com",
						port = 3307,
						user = "root",
						passwd = "123456")
		'''
		print "Connected"
	except :
		print "Can not connect to server"

if __name__ == '__main__':
	main()

File: perl/connect.pl
use strict;
use warning;
use DBI;

my $user = "root";
my $passwd = "123456";
my $host = "localhost";
my $database = "test";

my $dsn = "DBI:mysql:host=$host;database=$database ;";

my $dbh = DBI->connect($dsn,$user,$passwd) or die "Cannot connect to serve $host\n";
print "Connected\n";

$dbh->disconnect();
print "Disconnected\n";